src/main/java/seedu/clinkedin/logic/commands/DeleteRateCommand.java

```diff
@@ -18,7 +18,7 @@
  * Deletes rating of an existing person in the address book.
  */
 public class DeleteRateCommand extends Command {
-    public static final String COMMAND_WORD = "deleterate";
+    public static final String COMMAND_WORD = "deleteRate";
```

Comment:
Here the command word shall be `deleterate` only and not `deleteRate` because the parser allows the user input to be CASE INSENSITIVE so the parser first converts the user input to lower case and then checks whether it matches with the COMMAND_WORD or not.